solidjs-tailwind/src/contexts/RepoContext.jsx

```diff
@@ -0,0 +1,27 @@
+import { createEffect } from "solid-js";
+import { createSignal, createContext, useContext } from "solid-js";
+
+const RepoContext = createContext();
+
+export function RepoProvider(props) {
+  const [readme, setReadme] = createSignal({error: undefined,  text: undefined, isLoading: false});
+
+  createEffect(() => {
+    setReadme(props.readme)
+  });
+  
+
+  const repo = [
+    readme
+  ]
+
+  return (
+    <RepoContext.Provider value={repo}>
+      {props.children}
```

Comment:
try use the children solidjs provides use with.
https://www.solidjs.com/tutorial/props_children

Reply:
I don't think it is possible in the context, as i get this error below;

<img width="1476" alt="image" src="https://user-images.githubusercontent.com/36667083/204564824-1637ac74-9e4e-4b97-8bf0-7a69665c6bc5.png">

Also, this method of context I used was from their example guide